tests/terraform/checks/provider/awscc/test_credentials.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+
+import hcl2
+
+from checkov.terraform.checks.provider.awscc.credentials import check
+from checkov.common.models.enums import CheckResult
+
+
+class TestCredentials(unittest.TestCase):
+    def test_success_empty(self):
```

Comment:
Can you use the new testing format like this: https://github.com/bridgecrewio/checkov/blob/5e366b0107c5312fcc610f992dc49e5d245e9e06/tests/terraform/checks/module/registry/test_ModuleCheck.py

Reply:
Updated to use a reference file and assertions as in other examples.  The changes from main are merged in.